core/src/components/radio/test/basic/index.html

```diff
@@ -20,6 +20,26 @@
     </ion-header>
 
     <ion-content id="content" class="radio-test outer-content">
+      <ion-list>
```

Comment:
If you are on the "Fruits (Group w/ values)" radio group and you tab to the first element, should it uncheck the disabled radio when you use the arrow keys to go to the new radio?

Steps:

1. In the group, "Grape" should be disabled and checked.
2. Tab to group. "Apple" should be focused but unchecked.
3. Press the right arrow key. "Cherry" should be focused and now checked.

Reply:
I would say so, since that's how it works when there are no checked radios. 

1. Tab to the Pizza Toppings (Unchecked Group)
2. See that Pepperoni isn't selected
3. Use arrow keys, see that it selects

I think having a radio that is disabled and checked is really weird UX. They can't get back to the radio since it's disabled, so we just have to pretend it doesn't exist.